news/69.feature

```diff
@@ -0,0 +1 @@
+Add support for animated GIFs
```

Comment:
@reebalazs you credits are missing here.

Reply:
thanks, added